### ISSUE TITLE
AccessibilityResources.bundle not found

### ISSUE THREAD

**Issue by invyctus92** [OneSignal/react-native-onesignal#1156]
Hi everyone, I'm having some problems compiling a build for the stores while the build for the simulator works great.During the build I was having an issue with Multiple commands produce AccessibilityResources.bundle from React-Core.common-AccessibilityResources and React-Core.common-CoreModulesHeaders-AccessibilityResources. Deleting the first one solved the problem but after updating the library to version 4, it doesn't work anymore. Now I have this new problem in OneSignalNotificationServiceExtension.bundle where AccessibilityResources.bundle" not found

`Resource "/Users/Mario/Library/Developer/Xcode/DerivedData/Test-efwtcaprjxygkcchcfmphzivacqs/Build/Intermediates.noindex/ArchiveIntermediates/Test/BuildProductsPath/Release-iphoneos/React-Core.common/AccessibilityResources.bundle" not found. Run 'pod install' to update the copy resources script.`

Version IOS 
RN: 0.64.4
One-Signal: 4.0.3
Xcode 12.3

In podfile: 

```
target 'OneSignalNotificationServiceExtension' do
  pod 'OneSignal', '>= 2.9.3', '< 5.0'
end
```

NotificationService.m: 
```
#import <OneSignal/OneSignal.h>

#import "NotificationService.h"

@interface NotificationService ()

@property (nonatomic, strong) void (^contentHandler)(UNNotificationContent *contentToDeliver);
@property (nonatomic, strong) UNNotificationRequest *receivedRequest;
@property (nonatomic, strong) UNMutableNotificationContent *bestAttemptContent;

@end

@implementation NotificationService

- (void)didReceiveNotificationRequest:(UNNotificationRequest *)request withContentHandler:(void (^)(UNNotificationContent * _Nonnull))contentHandler {
    self.receivedRequest = request;
    self.contentHandler = contentHandler;
    self.bestAttemptContent = [request.content mutableCopy];
    
    [OneSignal didReceiveNotificationExtensionRequest:self.receivedRequest withMutableNotificationContent:self.bestAttemptContent];
    
    // DEBUGGING: Uncomment the 2 lines below and comment out the one above to ensure this extension is excuting
    //            Note, this extension only runs when mutable-content is set
    //            Setting an attachment or action buttons automatically adds this
    // NSLog(@"Running NotificationServiceExtension");
    // self.bestAttemptContent.body = [@"[Modified] " stringByAppendingString:self.bestAttemptContent.body];
    
    self.contentHandler(self.bestAttemptContent);
}

- (void)serviceExtensionTimeWillExpire {
    // Called just before the extension will be terminated by the system.
    // Use this as an opportunity to deliver your "best attempt" at modified content, otherwise the original push payload will be used.
    
    [OneSignal serviceExtensionTimeWillExpireRequest:self.receivedRequest withMutableNotificationContent:self.bestAttemptContent];
    
    self.contentHandler(self.bestAttemptContent);
}

@end
```

**Comment by Zurdge** [comment]
@invyctus92 
Do you remember how you solved this issue?
I'm getting the same thing pop up.
```
... BuildProductsPath/Release-iphoneos/React-Core.common/AccessibilityResources.bundle" not found. Run 'pod install' to update the copy resources script.
```
I've done the same as you and having issues with ```Multiple commands produce AccessibilityResources.bundle```

**Comment by solidtm** [comment]
Anyone solved this issue yet? I am having the same issue as well.

**Comment by Abdelaty** [comment]
any updates?
